ci/nightly/pipeline.template.yml

```diff
@@ -769,6 +769,25 @@ steps:
               composition: platform-checks
               args: [--scenario=DropCreateDefaultReplica, --execution-mode=parallel, "--seed=$BUILDKITE_JOB_ID"]
 
+      - id: checks-parallel-drop-create-default-replica-x86_64
```

Comment:
It's a bit annoying that this only works on x86-64 and we mostly run our tests on aarch64. Should we maybe move half of the platform-checks over to x86-64? Then we also have some coverage for upgrades etc.

Reply:
Yeah I was curious what your opinion was here. I'm happy to move half of them over, also happy to duplicate them entirely on x86? Not sure how many of self-hosted customers run x86 but it could be nice to have some move coverage there if so

Reply:
So far we have barely found any differences between x86-64 and aarch64, so I think duplicating is probably not worth it.